unit/BinaryPack.Unit/BinaryPack.Unit.csproj

```diff
@@ -1,12 +1,15 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
```

Comment:
Would update this to more modern .net like 7 (I don't have 3.1 installed anymore)